docs/releases/papers.md

```
@@ -1,64 +1,76 @@
 - **Related:** [research questions](../research/research_questions.md),
   [reproducibility policy](../research/reproducibility_policy.md), Rule R1/R7,
-  [Blueprint §12](../vision/MEDSCALE_STRATEGIC_BLUEPRINT_V1.md) (the four-paper arc)
+  [Blueprint §12](../vision/MEDSCALE_STRATEGIC_BLUEPRINT_V1.md) (the paper arc)
+
+This document is binding publication policy for any future paper/replication package.
+It is not evidence that experiments, result artifacts, papers, submissions, preprints,
+or replication packages currently exist or are authorized.
 
 ## The pipeline
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)